README.md

```diff
@@ -41,8 +41,8 @@ PORT=3001
 To get started, run the following commands:
 
 ```
-npm run init
-npm npm dev
+npm run init:dev
```

Comment:
ahh yeah, that was leftover from when I was going to be using both `init` and `init:dev`.  I decided to make it easier to just run by moving the production stuff into `npm start`

I'll move this in and then cleanup the mess.... You should be able to go back to `npm init` for dev when I'm done.